pkg/experiment/compactor/compaction_worker.go

```diff
@@ -51,7 +53,7 @@ type Config struct {
 func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	const prefix = "compaction-worker."
 	tempdir := filepath.Join(os.TempDir(), "pyroscope-compactor")
-	f.IntVar(&cfg.JobCapacity, prefix+"job-capacity", 3, "How many concurrent jobs will a compaction worker run at most.")
+	f.IntVar(&cfg.JobCapacity, prefix+"job-capacity", 1, "How many concurrent jobs will a compaction worker run at most.")
```

Comment:
In the current implementation, the `config.JobCapacity` parameters serves like a multiplier – how many workers to run per a core. I think that it may make sense to rename it: something like `job-concurrency` could be more precise